### PR TITLE
User profile page

### PR DESCRIPTION
**Pull request by Tiagod25**

### CODE REVIEW

File: web/src/containers/students/Student.js [hunk 1]
```diff
@@ -107,9 +114,20 @@ function Student(props) {
...
+              {matches === true && (
+                <Box>
+                  <img
+                    height="300px"
+                    width="300px"
+                    src={userInfo.profile_pic}
+                  />
+                </Box>
+              )}
+              {matches === false && (
```
Comment (pferreira101) [inline web/src/containers/students/Student.js]: Use a ternary expression, matches? (<Box>....</Box>) : (<Box>....</Box>)

Comment (Tiagod25) [reply web/src/containers/students/Student.js]: done

File: web/src/containers/UserProfile.js [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+  const onSubmit = () => {
+    var formData = new FormData();
+    formData.append("full_name", formik.values.full_name);
+    formData.append("address", formik.values.address);
+    formData.append("zip_code", formik.values.zip_code);
+    formData.append("short_bio", formik.values.short_bio);
+    formData.append("city", formik.values.city);
+    formData.append("country", formik.values.country);
+    UserService.updateInfo(formData);
+  };
```
Comment (Bishop19) [inline web/src/containers/UserProfile.js:50]: this function should be defined inside Formik (marked bellow)

File: web/src/containers/UserProfile.js [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+      address: "",
+      short_bio: "",
+      country: "",
+      image: "",
+      zip_code: "",
+      full_name: "",
+      city: "",
+      error: null,
+    },
+    onSubmit: async (values, { setFieldValue }) => {
```
Comment (Bishop19) [inline web/src/containers/UserProfile.js]: here

File: web/src/containers/UserProfile.js [hunk 4]
```diff
@@ -0,0 +1,208 @@
...
+  InputLabel,
+  Button,
+  Typography,
+} from "@material-ui/core";
+import { CloudUpload } from "@material-ui/icons";
+import { useFormik } from "formik";
+import { countries } from "../util/countries";
+import KycButton from "../components/KycButton";
+function UserProfile() {
+  useEffect(async () => {
```
Comment (Bishop19) [inline web/src/containers/UserProfile.js]: Is this working? useEffect can't handle async function as argument i guess (cf. other useEffects)

**Review by Bishop19 — CHANGES_REQUESTED** [review]

**Review by Bishop19 — APPROVED** [review]

**Review by joaop21 — APPROVED** [review]

**Pull request merged**